src/etl/presentation/api.py

```
@@ -0,0 +1,155 @@
+        else:
+            background_tasks.add_task(save_use_case.execute, request.source_path)
+
+        return {
+            "status": "accepted", 
+            "message": f"Загрузка из {request.source_type} запущена в фоновом режиме"
+        }
+    except HTTPException:
+        # Пробрасываем созданные нами HTTPException без изменений
+        raise
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)